### PR TITLE
Add survey flow

### PR DESCRIPTION
**Pull request by shchilkin**

### CODE REVIEW

File: web-client/pages/survey.tsx [hunk 1]
```diff
@@ -1,54 +1,104 @@
...
+    hobbies: number
+    social: number
+    health: number
+    mood: number
+    thoughts: string
+}
+
+const postForm = (data: DataModel): Promise<DataModel> => {
+    console.log('Send request')
+    return axios.post('http://10.100.27.14:8000/api/survey/', {
```
Comment (fpdotmonkey) [inline web-client/pages/survey.tsx:80]: do you need the protocol and base of the URL here?  ideally we shouldn't hardcode those

File: web-client/pages/survey.tsx [hunk 2]
```diff
@@ -71,60 +243,87 @@ export default function Survey() {
...
-                                onClick={() => dispatch(setConfigState(true))}
-                                colorTheme="red"
-                                href="/"
-                                text="Finish survey"
-                            />
+                            <Button
+                                onClick={() => {
+                                    postForm({
+                                        name: 'user',
+                                        sports: Number.parseInt(sports),
```
Comment (fpdotmonkey) [inline web-client/pages/survey.tsx:301]: should there be some handling for if somehow `sports` isn't `Number`-looking?  ie returning `-1` or something when `parseInt` returns `NaN`

**Review by fpdotmonkey — CHANGES_REQUESTED** [review]
Only the comment on the URL being hardcoded is important to me, otherwise this can be merged

**Review by nicksemin — APPROVED** [review]

File: web-client/pages/survey.tsx (hunk 1 above)
Comment (nicksemin) [reply web-client/pages/survey.tsx:80]: ideally lol :DDDDD

**Pull request merged**